Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -43,8 +44,9 @@ public FreezeOnMatchCustomization(Type targetType)
         /// that will be satisfied by the frozen specimen.
         /// </param>
         /// <exception cref="ArgumentNullException">
-        /// <paramref name="targetType"/> or<paramref name="matcher"/> is null.
+        /// <paramref name="targetType"/> or <paramref name="matcher"/> is null.
         /// </exception>
+        [Obsolete("Please use FreezeOnMatchCustomization(object, IRequestSpecification) instead.")]
```

Comment:
Wouldn't that also be the case if we deprecate this constructor and add `FreezeOnMatchCustomization(object, IRequestSpecification)`?